main.c

```diff
@@ -94,8 +96,14 @@ int main(void)
     plan_reset(); // Clear block buffer and planner variables
     st_reset(); // Clear stepper subsystem variables.
     progman_init();
-    report_revision(); // ADC - read revision voltage form revision voltage divider  
-  
+    systick_init();  // Init systick and systick callbacks
+
+    // Register first signals update callback
+    systick_register_callback(1000, signals_callback); // Start polling ADCs a second after init
```

Comment:
It was arbitrarily chosen. The SIGNALS_CALLBACK_INTERVAL is set to 20 ms, which is to short in this case - we need to wait for everything to be initialized before scheduling the first callback. A second may be a bit of an overkill.